src/Microsoft.PowerShell.Commands.Utility/resources/UtilityCommonStrings.resx

```diff
@@ -174,4 +174,10 @@
   <data name="TypeNotSupported" xml:space="preserve">
     <value>'{0}' is not supported in this system.</value>
   </data>
+  <data name="InvalidCharacterInParameter" xml:space="preserve">
+    <value>Parameter '{0}' contains at least one invalid character. Its value is: '{1}'</value>
+  </data>
+  <data name="CouldNotFindTemporaryFilename" xml:space="preserve">
```

Comment:
`Find` doesn't seem quite right, I would use `Create`, and change the message to `Could not create a temporary file name in {0}.`

Reply:
If we change the text we should rename `CouldNotFindTemporaryFilename` to `CouldNotCreateTemporaryFilename`.